CONTRIBUTING.md

```diff
@@ -63,7 +63,7 @@ Setup [`pre-commit`](https://pre-commit.com/) to automatically lint and format t
     make setup
     ```
 
-3. On push, the pre-commit hook will run. This runs `make format`, `make lint`, `UI linting` and `generation of helm docs`.
+3. On push, the pre-commit hook will run. This runs `make format`, `make lint` and `UI linting`.
```

Comment:
Should we add a note to the main README file and link to the new Helm charts repo fo where the charts can be found if someone wants to install?

Reply:
Added!